specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2021-04-01-preview/storagepool.json

```diff
@@ -516,13 +525,58 @@
           }
         },
         "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
         "x-ms-examples": {
           "Deallocate Disk Pool": {
             "$ref": "./examples/DiskPools_Deallocate.json"
           }
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StoragePool/{diskPoolName}/skus": {
```

Comment:
@j5lim is this a required API if we don't really support SKU updates today? Found this in[ RPC guidelines](https://github.com/Azure/azure-resource-manager-rpc/blob/2e1f2c11ce4cfdf309f9cc8ae71575cb8fb83319/v1.0/Addendum.md#enumerating-skus-for-an-existing-resource).

Reply:
@harshitha-akkaraju No, not required.

Reply:
Deleting this path because the RP won't support this operation in this version.